src/xmrEngine.js

```diff
@@ -837,20 +842,21 @@ class MoneroEngine {
         Object.assign({}, sendParams, {
           moneroSpendKeyPrivate: this.walletInfo.keys.moneroSpendKeyPrivate,
           onStatus: (code: number) => {
-            io.console.info(
-              `broadcastTx:SendFunds - onStatus:${code.toString()}`
-            )
+            this.log.warn(`broadcastTx:SendFunds - onStatus:${code.toString()}`)
           }
         })
       )
 
       edgeTransaction.txid = result.txid
       edgeTransaction.networkFee = result.networkFee
       edgeTransaction.txSecret = result.tx_key
-      io.console.info(`broadcastTx success txid: ${edgeTransaction.txid}`)
+      this.log.warn(
+        `broadcastTx success edgeTransaction ${JSON.stringify(edgeTransaction)}`
```

Comment:
Do not log the entire transaction object as it contains `otherParams.sendParams.moneroSpendKeyPrivate` . At minimum create a cleaner that only returns the parameters we want to show. As generally best practice, do no log full objects but only parameters you care about.

Reply:
As an extra security measure, please copy line 860 and put it below line 852 as the private key is no longer needed after the broadcast. Do this in a separate commit.